test/bin/grub-editenv

```diff
@@ -121,6 +164,9 @@ case $CMD in
 	"list")
 		variables_list "$@"
 		;;
+	"create")
```

Comment:
Do we use the `create` subcommand anywhere? We also don't define `unset`

Reply:
I have been using it in the ptest support script: Draft is here: https://github.com/KLSMartin/meta-rauc/blob/feature/integrate-ptest/recipes-core/rauc/files/run-ptest 

Note: There are still some bits missing.

I have also tried to run the qemu tests with the `grub-editenv` drop-in script, where the `create` is needed.